src/test/java/uk/gov/pay/api/it/PaymentsResourceGetIT.java

```diff
@@ -707,6 +707,20 @@ public void getPaymentWithAuthorisationSummaryThroughLedger() {
         assertPaymentWithAuthorisationSummary(response);
     }
 
+    @Test
+    public void getPaymentWithAuthorisationSummaryAndThreeDSecureFalseThroughLedger() {
```

Comment:
This name is inconsistent with the others (in terms of whether or not there’s `No` and `Required`).